enterprise/models.py

```diff
@@ -764,3 +775,82 @@ def __repr__(self):
         Return uniquely identifying string representation.
         """
         return self.__str__()
+
+
+@python_2_unicode_compatible
+class EnterpriseIntegratedChannel(TimeStampedModel):
+    """
+    Store information related to integrating with external enterprise systems.
+
+    EnterpriseIntegratedChannel is an external system that the Enterprise Customer uses to manage enterprise
+    related data. Each of these requires an edx plugin in order to send data to these systems, this model stores
+    information related to what and where to send data from the enterprise app.
+    """
+
+    name = models.CharField(max_length=255, blank=False, null=False, help_text=_("Third Party name."))
+    data_type = models.CharField(max_length=100, blank=False, null=False, help_text=_("Data Type"))
+    plugin_endpoint = models.CharField(
+        max_length=255,
+        blank=False,
+        null=False,
+        help_text=_("Internal Plugin endpoint.")
+    )
```

Comment:
Could you provide an example of the data that would get stored using this model?